enterprise/conf/locale/en/LC_MESSAGES/django.po

```diff
@@ -1215,19 +1215,3 @@ msgstr ""
 #, python-brace-format
 msgid "A user with the username {username} was not found."
 msgstr ""
-
-#: integrated_channels/integrated_channel/models.py:30
-msgid "Third Party name."
-msgstr ""
-
-#: integrated_channels/integrated_channel/models.py:31
-msgid "Data Type"
-msgstr ""
-
-#: integrated_channels/integrated_channel/models.py:35
-msgid "Enterprise Integrated Channel"
-msgstr ""
-
-#: integrated_channels/integrated_channel/models.py:36
-msgid "Enterprise Integrated Channels"
-msgstr ""
```

Comment:
So looks like someone already got to fixing the missing translations before me, @itsjeyd, so those additions are gone.